sdk/core/core-tracing/src/tracers/test/testSpan.ts

```diff
@@ -64,7 +65,7 @@ export class TestSpan extends NoOpSpan {
    * @param startTime The startTime of the event (defaults to now)
    */
   constructor(
-    parentTracer: TestTracer,
+    parentTracer: TestTracerLike,
```

Comment:
I don't think we need `TestTracerLike` at all, we can just use `Tracer` here, right?

Reply:
I'll try

Reply:
I couldn't remove the TestTracer in full, but I was able to update this to Tracer.